src/core/server/saved_objects/service/lib/update_objects_spaces.ts

```diff
@@ -260,14 +292,32 @@ export async function updateObjectsSpaces({
     ? await client.bulk({ refresh, body: bulkOperationParams, require_alias: true })
     : undefined;
 
+  // Delete aliases if necessary, ensuring we don't have too many concurrent operations running.
+  const mapper = async ({ type, id, namespaces, deleteBehavior }: ObjectToDeleteAliasesFor) =>
+    deleteLegacyUrlAliases({
+      mappings,
+      registry,
+      client,
+      getIndexForType,
+      type,
+      id,
+      namespaces,
+      deleteBehavior,
+    }).catch((err) => {
+      // The object has already been unshared, but we caught an error when attempting to delete aliases.
+      // A consumer cannot attempt to unshare the object again, so just log the error and swallow it.
+      logger.error(err.message);
```

Comment:
Should this error have more context?

```suggestion
      logger.error(`Failed to delete legacy URL aliases: ${err.message}`);
```

Reply:
The error originates from `deleteLegacyUrlAliases`, and those error messages are pretty detailed.
Are you suggesting we should change this message to mention that this is happening during `updateObjectsSpaces`?

Reply:
> The error originates from `deleteLegacyUrlAliases`, and those error messages are pretty detailed. Are you suggesting we should change this message to mention that this is happening during `updateObjectsSpaces`?

I agree that the _expected_ error messages are pretty detailed, and there is currently little room for unexpected errors to occur the way that `deleteLegacyUrlAliases` is constructed. Yeah I think mentioning that this is happening during `updateObjectsSpaces` would be sufficient. It gives us another piece of "grepable" text in the error message should we ever have to trace this in production

Reply:
Great, done for both consumers in ed54e64b5ac51c47c7b0e79eed851909c5a02aec